com.unity.render-pipelines.universal/Runtime/ForwardRenderer.cs

```diff
@@ -94,6 +94,8 @@ private static class Profiling
 
         public ForwardRenderer(ForwardRendererData data) : base(data)
         {
+            UniversalRenderPipelineAsset urpAsset = GraphicsSettings.renderPipelineAsset as UniversalRenderPipelineAsset;
```

Comment:
Shouldn't it be GraphicsSettings.currentPipelineAsset? i'm actually confused about the difference between this and renderPipelineAsset

Reply:
I took it from line 140, not sure